### ISSUE TITLE
Treasury/Auction Account

### ISSUE THREAD

**Issue by Mike-E-angelo** [StarbeamOne/home#97]
Getting money in and out is a known/obvious fraud risk, one that Stripe/PayPal/Coinbase should be able to handle most if not all the time.  But what happens when that money is already "parked" in our domain?  We have to reverse/cover the activity.  If money has been used to already purchase licenses, we can sell them to a treasury/auction account just as they do for police departments for assets involved in stolen/criminal/fraudulent activity.

**Comment by wrexbe** [comment]
After buying it, it could be re-sold to get money back into the Treasury account for future issues

**Comment by Mike-E-angelo** [comment]
> After buying it, it could be re-sold to get money back into the Treasury account for future issues

Indeed, I left that part out as it is the presumed outcome of the arrangement.  Thank you for clarifying. 🙏
